src/main/java/ch/njol/skript/aliases/ItemData.java

```diff
@@ -607,7 +607,8 @@ public ItemData aliasCopy() {
 					data.itemFlags |= ItemFlags.CHANGED_TAGS;
 				data.stack.setItemMeta(meta);
 			}
-			ItemUtils.setDamage(data.stack, 0); // Set to undamaged
+			if (ItemUtils.getDamage(stack) > 0)
+				ItemUtils.setDamage(data.stack, 0); // Set to undamaged iff item is damaged
```

Comment:
```suggestion
				ItemUtils.setDamage(data.stack, 0); // https://github.com/SkriptLang/Skript/issues/7687
```

Reply:
I'm going to add the issue link, but rather to the line above since that is the change itself